src/main/python/main/ayab/knitprogress.py

```diff
@@ -124,8 +132,6 @@ def update_progress(self, status: Status) -> None:
         if self.columnCount() != len(columns):
             self.setColumnCount(len(columns))
         n_cols = len(columns)
-        if n_cols < 4:
-            self.hideColumn(5)
```

Comment:
Removing those two lines (which do look like dead code, as the number of columns is synchronized with the size of the `columns` array earlier, so column 5 can't exist if `len(columns) < 4`) makes the `n_cols` assignment above obsolete as well.